satpy/etc/enhancements/generic.yaml

```diff
@@ -3,9 +3,13 @@
 enhancements:
   default:
     operations:
-    - name: stretch
-      method: !!python/name:satpy.enhancements.contrast.stretch
-      kwargs: {stretch: linear}
+      - name: warn_no_predefined
+        method: !!python/name:satpy.enhancements.contrast.noop_with_warning
+        kwargs:
+          msg: "No YAML enhancement configured for {name!r}. Falling back to a dynamic linear stretch."
+      - name: stretch
+        method: !!python/name:satpy.enhancements.contrast.stretch_if_not_integer
+        kwargs: {stretch: linear}
```

Comment:
Do we want `linear` or `crude` as the default stretch? `linear` has the cutoffs, `crude` stretches between min and max values. Meh, I guess the current `linear` is fine to get rid of the outliers.

Reply:
Doesn't crude require defining numeric values?  That wouldn't work well here because there is no way to define values that work by default.

Reply:
Agreed. It'd be nice to use `crude` for performance but we just can't.